frontend/package.json

```diff
@@ -1,38 +1,47 @@
 {
-	"name": "frontend",
-	"version": "0.1.0",
-	"private": true,
-	"dependencies": {
-		"axios": "^0.18.0",
-		"prop-types": "^15.6.2",
-		"react": "^16.6.0",
-		"react-dom": "^16.6.0",
-		"react-redux": "^5.1.0",
-		"react-router-dom": "^4.3.1",
-		"react-scripts": "2.1.1",
-		"redux": "^4.0.1",
-		"redux-thunk": "^2.3.0",
-		"styled-components": "^4.0.3"
-	},
-	"scripts": {
-		"start": "react-scripts start",
-		"build": "react-scripts build",
-		"test": "react-scripts test --env=jsdom --verbose",
-		"eject": "react-scripts eject"
-	},
-	"eslintConfig": {
-		"extends": "react-app"
-	},
-	"browserslist": [
-		">0.2%",
-		"not dead",
-		"not ie <= 11",
-		"not op_mini all"
-	],
-	"devDependencies": {
-		"enzyme": "^3.7.0",
-		"enzyme-adapter-react-16": "^1.6.0",
-		"react-test-renderer": "^16.6.1",
-		"redux-logger": "^3.0.6"
-	}
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^0.18.0",
+    "prop-types": "^15.6.2",
+    "react": "^16.6.0",
+    "react-dom": "^16.6.0",
+    "react-redux": "^5.1.0",
+    "react-router-dom": "^4.3.1",
+    "react-scripts": "2.1.1",
+    "redux": "^4.0.1",
+    "redux-thunk": "^2.3.0",
+    "styled-components": "^4.0.3"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom --verbose",
+    "eject": "react-scripts eject",
+    "precommit": "pretty-quick --staged"
```

Comment:
I would argue for a `--verbose` flag on the precommit script as well, in case prettier bails out with an error.